src/scripts/main.js

```diff
@@ -1,3 +1,38 @@
 'use strict';
 
-// write code here
+const tableHeader = document.querySelector('tr');
+const tableBody = document.querySelector('tbody');
+const toNumber = (string) => {
+  if (typeof string === 'string') {
```

Comment:
The 'toNumber' function accepts a string, but there is no check for an empty string. If an empty string is passed, 'parseInt' will return NaN. Consider adding validation for an empty string.